projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php

```diff
@@ -203,6 +203,13 @@ public function add_options_menu( $wp_admin = false ) {
 		add_options_page( esc_attr__( 'Hosting Configuration', 'jetpack' ), __( 'Hosting Configuration', 'jetpack' ), 'manage_options', 'https://wordpress.com/hosting-config/' . $this->domain, null, 6 );
 
 		parent::add_options_menu( $wp_admin );
+
+		// No need to add a menu linking to WP Admin if there is already one.
+		if ( ! $wp_admin ) {
+			$parent_menu_slug = $wp_admin ? 'options-general.php' : 'https://wordpress.com/settings/general/' . $this->domain;
```

Comment:
We are already checking `$wp_admin` is false, so we can simplify this to
```suggestion
			$parent_menu_slug = 'https://wordpress.com/settings/general/' . $this->domain;
```

Reply:
Yep. Moved that into the condition and forgot to remove that check. Done